aws/resource_aws_codebuild_project_test.go

```diff
@@ -350,8 +365,27 @@ func testAccCheckAWSCodeBuildProjectDestroy(s *terraform.State) error {
 	return fmt.Errorf("Default error in CodeBuild Test")
 }
 
-func testAccAWSCodeBuildProjectConfig_basic(rName string) string {
+func testAccAWSCodeBuildProjectConfig_basic(rName string, subnet string) string {
```

Comment:
Same with the mention above about the test itself, the VPC configuration should be separated as well. Thanks!